cpp/include/cudf/unary.hpp

```diff
@@ -93,5 +93,37 @@ std::unique_ptr<cudf::column> is_valid(cudf::column_view const& input,
 std::unique_ptr<column> cast(column_view const& input, data_type out_type,
                              rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Creates a column of `BOOL8` elements.
```

Comment:
Needs to be a bit more descriptive. 
```suggestion
 * @brief Creates a column of `BOOL8` elements indicating the presence of `NaN` values
 * in a column of floating point values. 
```